flytekit/interactive/constants.py

```diff
@@ -0,0 +1,9 @@
+# Default max idle seconds to terminate the flyteinteractive server
+HOURS_TO_SECONDS = 60 * 60
+MAX_IDLE_SECONDS = 10 * HOURS_TO_SECONDS  # 10 hours
+
+# Subprocess constants
+EXIT_CODE_SUCCESS = 0
+
+# Set it to True to run vscode server when the task
+FLYTE_ENABLE_VSCODE_KEY = "FLYTE_ENABLE_VSCODE"
```

Comment:
Can we shorten the value? Take a look at https://github.com/flyteorg/flyte/issues/5665 for more details. In other words:

```suggestion
FLYTE_ENABLE_VSCODE_KEY = "_F_E_VS"
```

Reply:
I updated it, thanks. I'll update the backend and flyteconsole as well.